saleor/graphql/core/types/common.py

```diff
@@ -57,6 +57,10 @@ class ProductError(Error):
     code = ProductErrorCode(description="The error code.")
 
 
+class BulkProductError(ProductError):
+    index = graphene.Int(description="Input list index with error")
```

Comment:
```suggestion
    index = graphene.Int(description="The index that triggered the error")
```

Reply:
"Index of an input list item that caused the error"